protos/docs/datacatalog/index.rst

```diff
@@ -1,10 +1,10 @@
 Flyte Data Catalog Service
-=================================
+============================
 
-These protos provide the interface definition for the Data Catalog Service. Data Catalog is a service for
+Protos provide the interface definition for the Data Catalog Service. Data Catalog is a service for
```

Comment:
```suggestion
Protos provides the interface definition for the Data Catalog Service. Data Catalog is a service for
```

Reply:
It should be "provide" only, right @AlekhyaSasi?